week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,26 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  if (isNumber(num) && isPositive(num) && isLessThanOrEqual_100(num))
+    return true;
+  else return false;
```

Comment:
This does technically work, but 99% of the time we want to include the curly braces when using an `if` or `else` statement.

Reply:
Thank you Ali, this has now been updated